### PR TITLE
bug #243: add debug log if no testng found on classpath

### PR DESCRIPTION
**Pull request by missedone**
hi @cbeust 
I'm not able to reproduce the issue #243 
so here is my plan, in this PR, i added some debug log for further diagnose.
i'd like to merge this onto maser (aka. 6.9.12-SNAPSHOT), then ask somebody to reproduce the issue.
hopefully, with the generated log i can figure out the cause.

regards to the release, as this branch is created from 6.9.11 tag, the fix for #243 will be on this branch, so once it's fixed (hopefully won't take too long), we can release another 6.9.11 plugin from this branch.

what do you think?
thanks

### CODE REVIEW

File: testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java [hunk 1]
```diff
@@ -344,6 +341,8 @@ protected VMRunnerConfiguration createVMRunner(
 
     Version testngVer = getRuntimeTestNGVersion(classpathList);
     if (testngVer == null) {
+      TestNGPlugin.log(TestNGPlugin.createError("Can't recognize runtime TestNG version from classpath: "
```
Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: log the classpath for further diagnose

File: testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java [hunk 2]
```diff
@@ -391,35 +390,25 @@ private Version getRuntimeTestNGVersion(List<String> classpath)
...
+            return new Version(mainAttrs.getValue("Specification-Version"));
           }
         } catch (Exception e) {
           TestNGPlugin.log(e);
         }
 
-        if (ver != null) {
-          return ver;
-        }
+        TestNGPlugin.log(cp + " was the potential target, but no canonical version found in MANIFEST.MF");
```
Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: if the jar file name starts with 'testng', it's pretty much likely be the testng.jar, if we can't parse the version from its MANIFEST.MF, we log the file name for further diagnose.

**Comment by cbeust** [comment]
Sounds good to me. How do you want to let uses download this snapshot? Do you want me to upload it to `http://beust.com/eclipse-old` and then we can ask users to point to the correct update site?

**Comment by missedone** [comment]
i will ask them to install from the beta update-site

**Pull request merged**